### PR TITLE
implement #544

### LINKED ISSUE #544: Way to get all elements in a list except for an element at a specified index
Assuming the command is called `x`: and lists can be declared with `[]`:

```
[1,2,3]0x => [2, 3]
[1,2,3]1x => [1,3]
[1,2,3,1]3x => [1,2,3]
[1,2,3,1]0x => [2,3,1]
```

### PR DESCRIPTION
**Pull request by Lyxal**
closes #544

### CODE REVIEW

File: documents/knowledge/elements.yaml [hunk 1]
```diff
@@ -65,17 +65,17 @@
...
 
 - element: "⟇"
-  name: Reversed Logical Or
+  name: Remove at Index
   arity: 2
-  description: Returns  the same as logical or but with reversed arguments.
+  description: Returns every item in a list except the item at the given index.
   overloads:
-    any-any: b or a
+    any-num: Remove item b of a
```
Comment (cgccuser) [inline documents/knowledge/elements.yaml:72]:
```suggestion
    any-num: Remove item b of a
    num-any: Remove item a of b
```

File: documents/knowledge/elements.yaml [hunk 2]
```diff
@@ -65,17 +65,17 @@
...
   vectorise: false
   tests:
-    - "[0, 0] : 0"
-    - '["", 1] : 1'
-    - "[[1,2,3], 0] : [1,2,3]"
-    - "[1, 2] : 2"
+    - "[[1,2,3], 0] : [2,3]"
+    - "[[1,2,3], 1] : [1,3]"
+    - "[[1,2,3,1], 3] : [1,2,3]"
+    - "[[1,2,3,1], 0] : [2,3,1]"
```
Comment (cgccuser) [inline documents/knowledge/elements.yaml:78]:
```suggestion
    - "[3, [1,2,3,1]] : [1,2,3]"
    - "[0, [1,2,3,1]] : [2,3,1]"
```
imma just swap these around so we can confirm that the second overload works

**Review by cgccuser — CHANGES_REQUESTED** [review]
LGTM

**Review by cgccuser — APPROVED** [review]
I love being able to make suggestions from here

**Review by pxeger — APPROVED** [review]

**Review by TheSecondComing123 — COMMENTED** [review]
Looks good to me.

**Pull request merged**